American Whitewater/ViewControllers/Filtering/FilterViewController.swift

```diff
@@ -0,0 +1,516 @@
+import UIKit
+import CoreLocation
+
+class FilterViewController: UIViewController, UITableViewDelegate, UITableViewDataSource, UISearchBarDelegate, CLLocationManagerDelegate,
+                            UICollectionViewDelegate, UICollectionViewDelegateFlowLayout, UICollectionViewDataSource {
+    
+    @IBOutlet weak var contentCollectionView: UICollectionView!
+    
+    @IBOutlet weak var filterSegmentControl: UISegmentedControl!
+       
+    let locationManager = CLLocationManager()
+    let geoCoder = CLGeocoder()
+    
+    var selectedRegions:[Region] = []
+    var selectedRegionsOrig:[Region] = []
+    var usaRegions = Region.states
+    var internationalRegions = Region.international
+    var classFilters:[Int] = DefaultsManager.classFilter
+
+    @IBOutlet weak var regionsContainerView: UIView!
+        
+    var regionsTableViewRef: UITableView?
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+     
+        
```

Comment:
Minor: Seems like there's generally a lot of extra whitespace, and some inconsistency with whether to end a function with a blank line or not.

Reply:
yep, I have no preference (as you can see) and this was never stated or set. Just need to know what to start doing and I can start fixing it when we're in the code.

Reply:
Yeah I'd say just pick something and stick to it.